### PR TITLE
Importing accounts

### PR DESCRIPTION
**Pull request by kosssi**
Le but de cette PR est d'affiche un écran spécifique sur la page Solde Actuel avec les comptes entrain d'être importé.
On peut voir une démo [ici](https://testbanksimportingaccounts-banks.cozy.works/#/balances). 

Il me reste 3 choses à faire:
- [x] Supprimer le clique sur les `ExpansionPanel`
- [x] Vérifier une traduction du bouton du bas qui est en anglais (Add a Bank)...
- [x] Implémenter la fonction all pour TriggerCollection dans cozy-client [ici](https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/TriggerCollection.js#L32)

Vu qu'il n'est pas possible d'injecter des fixtures j'ai ajouté directement dans le code ceci:
```js
    const triggers = [
      {
        _id: 'd8690a8947ebc934f81df23ec52395a1',
        domain: 'gangsters.cozy.rocks',
        prefix: 'gangsters.cozy.rocks',
        type: '@cron',
        worker: 'konnector',
        arguments: '0 52 2 * * *',
        debounce: '',
        options: null,
        message: {
          konnector: 'societegenerale',
          banking: true,
          account: 'e02702daad059578a50cf9f78e24ebc8'
        }
      },
      {
        _id: 'd8690a8947ebc934f81df23ec52395a1',
        domain: 'gangsters.cozy.rocks',
        prefix: 'gangsters.cozy.rocks',
        type: '@cron',
        worker: 'konnector',
        arguments: '0 52 2 * * *',
        debounce: '',
        options: null,
        message: {
          konnector: 'bnpparibas82',
          banking: true,
          account: 'e02702daad059578a50cf9f78e24ebc8'
        }
      }
    ]
```

### CODE REVIEW

File: src/ducks/balance/Balance.jsx [hunk 1]
```diff
@@ -123,19 +128,21 @@ class Balance extends PureComponent {
...
-    if (
-      isCollectionLoading(accountsCollection) ||
-      isCollectionLoading(groupsCollection)
-    ) {
+    const collections = [
+      accountsCollection,
+      groupsCollection,
+      triggersCollection
+    ]
+    if (some(collections, isCollectionLoading)) {
```
Comment (drazik) [inline src/ducks/balance/Balance.jsx]: Nit: I would use [Array.prototype.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) here as it's the same thing, but native:

```suggestion
    if (collections.some(isCollectionLoading)) {
```

File: src/ducks/balance/components/AccountsImporting.jsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          </Padded>
+        </VerticalBox>
+      </Header>
+      <Padded className="u-pt-0">
+        <BalancePanels
+          groups={groups}
+          panelsState={panelsState}
+          warningLimit={0}
+          onSwitchChange={emptyFct}
+          onPanelChange={emptyFct}
```
Comment (drazik) [inline src/ducks/balance/components/AccountsImporting.jsx]: What about checking in `BalancePanels` if the props is a function? So if we pass nothing, it's not called. It avoids an empty functions + function calls to this empty functions.

File: src/ducks/balance/components/AccountsImporting.styl [hunk 3]
```diff
@@ -0,0 +1,24 @@
+@require 'settings/breakpoints'
+
+$header-margin-bottom = -28px
+
+.content
+    height 245px
+    color white
```
Comment (drazik) [inline src/ducks/balance/components/AccountsImporting.styl]: Can you use a CSS variable here? Not necesarily a new one, but something like `primaryContrastTextColor`?

File: src/ducks/balance/components/GroupPanel.jsx [hunk 4]
```diff
@@ -154,7 +154,7 @@ class GroupPanel extends React.PureComponent {
             disableRipple: true
           }}
           className={cx({
-            [styles['GroupPanelSummary--unchecked']]: !checked
+            [styles['GroupPanelSummary--unchecked']]: !checked && !group.loading
```
Comment (drazik) [inline src/ducks/balance/components/GroupPanel.jsx]: Maybe this class name is not relevant anymore if it's used not only to show that a panel is unchecked, but also when it's loading.

File: src/locales/en.json [hunk 5]
```diff
@@ -65,7 +65,6 @@
     "solde": "Balance",
     "account_number": "Account number",
     "account_name": "Account name",
-    "add_bank": "Add a bank",
```
Comment (drazik) [inline src/locales/en.json]: It was already at the right place in french translations?

File: src/ducks/balance/components/AccountRowLoading.jsx [hunk 6]
```diff
@@ -8,7 +9,7 @@ import styles from './AccountRow.styl'
 import stylesLoading from './AccountRowLoading.styl'
 
 const AccountRowLoading = ({ t, institutionSlug }) => (
-  <li className={styles.AccountRow}>
+  <li className={cx(styles.AccountRow, stylesLoading.noLink)}>
```
Comment (drazik) [inline src/ducks/balance/components/AccountRowLoading.jsx]: The base component should have the `u-clickable` class, and this one should remove it. This way we don't add a class for this purpose.

File: src/locales/en.json (hunk 5 above)
Comment (kosssi) [reply src/locales/en.json]: yes I clean it [here](https://github.com/cozy/cozy-banks/commit/891b25b54897dfd4fd20911cfda9ec7ea80403e8). I will fixup this commit ;)

File: src/ducks/balance/components/GroupPanel.jsx (hunk 4 above)
Comment (kosssi) [reply src/ducks/balance/components/GroupPanel.jsx]: I create a variable `const isUncheckable = !group.loading` which allow to keep a coherence

**Pull request merged**